pkg/reconciler/route/resources/certificate.go

```diff
@@ -58,6 +60,9 @@ func MakeCertificates(route *v1alpha1.Route, domainTagMap map[string]string) []*
 				Name:            certName,
 				Namespace:       route.Namespace,
 				OwnerReferences: []metav1.OwnerReference{*kmeta.NewControllerRef(route)},
+				Annotations: map[string]string{
```

Comment:
Should we in general propagate the annotations/labels from the Route?

Reply:
SGTM as we have already done this for ClusterIngress.